README.md

```diff
@@ -51,8 +68,36 @@ core guidelines that govern how things work inside ELGarash.
 
 11. Add references as the final section of the report.
 
-12. No Experiments should go un-reviewed, open a PR, and someone else should review, sign-off, and approve your experiment.
+12. No Experiment should go un-reviewed, open a PR, and someone else should review, sign-off, and approve your experiment.
+
+13. No direct pushes to main.
+
+14. Even if the experiment fails, it should be reported.
+
+# How to write?
+
+1. Keep your writings short and persuasive, and don’t use extra words → Hemingway app can help you with that.
+
+2. **_Magician’s Mar Magician’s Mark_**: Whenever you feel like writing “due to the fact that,” “as a result,” “therefore,” “for that reason,” “accordingly,” or even “because,” consider using a colon instead to make the reader ask “why”:
```

Comment:
Do you mean **_Magician’s Mark_** instead of **_Magician’s Mar Magician’s Mark_**
The [document](https://www2.nycbar.org/htmlemail/YLC/Five%20Ways%20to%20Write%20Like%20The%20Economist.pdf) you mentioned

Reply:
Yup

```suggestion
2. **_Magician’s Mark_**: Whenever you feel like writing “due to the fact that,” “as a result,” “therefore,” “for that reason,” “accordingly,” or even “because,” consider using a colon instead to make the reader ask “why”:

```